blog/2016-12-05-easy-upgrades-processed-by-git.md

```diff
@@ -8,11 +8,54 @@ authorTwitter: ncuillery
 category: announcements
 ---
 
-Each release of React Native may come with changes inside the iOS and Android sub-projects that you have to report in your project. This synchronisation has always been a major pain point because of the poor changes detection within your source files. Today, I'm proud to announce a new upgrading process relying on the most popular version control system: Git.
+Upgrading to each new version of React Native has been painful. You might have seen something like this before:
 
-The key concept in this operation is the generation of a Git patch that contains all the changes required by React Native from your current version to the requested one.
+```
+$ react-native upgrade
+Upgrading project to react-native v0.38.0
+  conflict ios/MyAwesomeApp/AppDelegate.m
+? Overwrite ios/MyAwesomeApp/AppDelegate.m? (Ynaxdh)
+  y) overwrite
+  n) do not overwrite
+  a) overwrite this and all others
+  x) abort
+  d) show the differences between the old and the new
+  h) Help, list all options
+ ```
+
+None of the options is what we want. By overwriting the file we lose changes we've made locally. What if we tried to resolve the conflicts automatically instead?
+
+Today I am proud to introduce a new tool which makes upgrading easier. The tool is called `react-native-git-upgrade` and uses Git behind the scenes.
```

Comment:
Leave out the part Git is the most popular. It probably is but I'm not 100% sure and it's not important in this blogpost.